src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectPropertyDescriptor.cs

```diff
@@ -298,12 +298,8 @@ private MethodInfo GetMethodValue
                     {
                         if (_propInfo == null)
                         {
-#if VERIFY_REFLECTION_CHANGE
```

Comment:
Since both this fix and the previous one you submitted were related to missing compilation constants, I looked for all #ifs in this component, and see that there are two others: FEATURE_SKIP_INTERFACE and FEATURE_COMPONENT_COLLECTION.  Are either of those causing problems, too?

Reply:
Possibly. I have no idea what impact `FEATURE_SKIP_INTERFACE` has (if any). It looks like `FEATURE_COMPONENT_COLLECTION` allows collecting property extenders from a component site's container if `IExtenderListService` is not present. However, for our designer scenarios, `IExtenderListService` _is_ present, so I wouldn't expect us to head that disabled code path.